src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -350,6 +350,53 @@ enable_fencing=true
 
 [Storage back-end Settings]
```

Comment:
backend

Reply:
This will require a bunch of code changes